tests/storages_tests/rdb_tests/test_with_server.py

```diff
@@ -0,0 +1,98 @@
+from multiprocessing import Pool
+import os
+from typing import Sequence
+from typing import Tuple
+
+import numpy as np
+import pytest
+
+import optuna
+
+_STUDY_NAME = "_test_multiprocess"
+
+
+def f(x: float, y: float) -> float:
+    return (x - 3) ** 2 + y
+
+
+def objective(trial: optuna.Trial) -> float:
+    x = trial.suggest_float("x", -10, 10)
+    y = trial.suggest_float("y", -10, 10)
+    trial.report(x, 0)
+    trial.report(y, 1)
+    trial.set_user_attr("x", x)
+    trial.set_system_attr("y", y)
+    return f(x, y)
+
+
+def run_optimize(args: Tuple[str, str]) -> None:
+    study_name = args[0]
+    storage = args[1]
+    # Create a study
+    study = optuna.create_study(study_name=study_name, storage=storage, load_if_exists=True,)
+    # Run optimization
+    study.optimize(objective, n_trials=20)
+
+
+@pytest.fixture
+def storage() -> str:
```

Comment:
Might be controversial so you can ignore it but I personally think that the following increases the readability.
```suggestion
def storage_url() -> str:
```

Reply:
Thank you for your suggestion. Applied!